openapi/nexus.json

```diff
@@ -6007,25 +6067,25 @@
           {
             "type": "object",
             "properties": {
-              "src": {
-                "type": "string"
-              },
               "type": {
                 "type": "string",
                 "enum": [
                   "url"
                 ]
+              },
+              "url": {
+                "type": "string"
               }
```

Comment:
The type here serialized a bit weird. This is an enum with the uuid below being a string. I assume they'll generate out to both being the same, but the difference is still peculiar.